ui/pages/main_page.py

```diff
@@ -35,3 +96,104 @@ def click_explore_developer_hub_button(self):
 
     def click_add_your_dapp_button(self):
         self.wait.until(EC.presence_of_element_located(MainPage.add_your_dapp_button)).click()
+
+    def input_email(self):
+        email_field = self.wait.until(EC.visibility_of_element_located(MainPage.email_input_field))
+        email_field.send_keys(MainPage.email)
+
+    def check_already_subscribed_text(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.subscription_notification_text)).is_displayed()
+
+    def click_ask_me_later_button(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.ask_me_later_button)).click()
+
+    def click_accept_button(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.accept_button)).click()
+
+    def assert_cookie_banner_is_invisible(self):
+        self.wait.until(EC.invisibility_of_element(MainPage.cookie_banner))
+
+    def assert_cookie_banner_is_visible(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.cookie_banner)).is_displayed()
+
+    def click_subscribe_button(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.subscribe_button)).click()
+
+    def click_social_network_icon(self, icon_name):
+        self.wait.until(EC.visibility_of_element_located(icon_name)).click()
+
+    def click_terms_of_use_link(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.terms_of_use_link)).click()
+
+    def click_disclaimer_link(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.disclaimer_link)).click()
+
+    def click_cookie_policy_link(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.cookie_policy_link)).click()
+
+    def click_privacy_policy_link(self):
+        self.wait.until(EC.visibility_of_element_located(MainPage.privacy_policy_link)).click()
+
+    def check_page_title(self, page_title):
+        self.wait.until(EC.visibility_of_element_located(page_title)).is_displayed()
+
+    def check_news_section(self):
+        news_count = len(self.wait.until(EC.visibility_of_all_elements_located(MainPage.news_section)))
+        assert news_count == 3, f"Number of news is {news_count}"
```

Comment:
don't hard code expected values
```suggestion
    def check_news_section(self, count: int):
        news_count = len(self.wait.until(EC.visibility_of_all_elements_located(MainPage.news_section)))
        assert news_count == count, f"Number of news is {news_count}"
```

Reply:
Fixed